templates/web/nottinghamshirepolice/report/new/councils_text_all.html

```diff
@@ -0,0 +1,18 @@
+[% category = mark_safe(category) %]
+
+<p>
+[% UNLESS non_public_categories.$category;
+
+    tprintf(
+        loc('These will be sent to the <strong>Immediate Justice Team</strong> and also published online for others to see, in accordance with our <a href="%s">privacy policy</a>.'),
+        c.cobrand.privacy_policy_url
+    );
+
+ELSE;
+
+    loc('These will be sent to the <strong>Immediate Justice Team</strong> but not published online.');
+
+END %]
+
+[% TRY %][% INCLUDE 'report/new/councils_extra_text.html' %][% CATCH file %][% END %]
```

Comment:
Presumably we don't have this template for this cobrand so don't need this, but no harm leaving it in.